frontend/src/components/SingleStore.js

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+import { useEffect, useState } from "react";
+import styled from "styled-components/macro";
+import FontSizes from "../constants/FontSizes";
+import getProductsByStore from "../services/getProductsByStore";
+import getStoreById from "../services/getStoreById";
+import Product from "./Product";
+
+export default function SingleStore() {
+  const storeLink = window.location.href;
+  const pos = storeLink.lastIndexOf("/");
+  const storeId = storeLink.substr(pos + 1);
+
+  const [storeName, setStoreName] = useState([]);
+  const [products, setProducts] = useState([]);
+  const [meat, setMeat] = useState([]);
+  const [milk, setMilk] = useState([]);
+  const [cheese, setCheese] = useState([]);
+
+  useEffect(() => {
+    getStoreById(storeId)
+      .then((data) => setStoreName(data))
+      .catch((error) => console.log(error.message));
+  }, []);
+
+  useEffect(() => {
+    getProductsByStore(storeId)
+      .then((data) => setProducts(data))
+      .catch((error) => console.log(error.message));
```

Comment:
console.error

Reply:
kannst du beiden useEffects ja auch in einen packen - haben die selbe Abhängigkeit []